Draw.it.Server/Repositories/WordPool/IWordPoolRepository.cs

```diff
@@ -0,0 +1,14 @@
+using Draw.it.Server.Models.WordPool;
+
+namespace Draw.it.Server.Repositories.WordPool
+{
+    public interface IWordPoolRepository
+    {
+        CategoryModel GetCategoryById(long categoryId);
+        IEnumerable<CategoryModel> GetAllCategories();
+        IEnumerable<WordModel> GetAllWords();
+        IEnumerable<WordModel> GetWordsByCategoryId(long categoryId);
```

Comment:
```suggestion
        CategoryModel? FindCategoryById(long categoryId);
        IEnumerable<CategoryModel> GetAllCategories();
        IEnumerable<WordModel> GetAllWords();
        IEnumerable<WordModel> FindWordsByCategoryId(long categoryId);
```
Repositories should softly "find", services - try to "get" (and throw an error if unsuccessful). That is the pattern we stick to in this project